components/brave_rewards/browser/database/database_recurring_tip.cc

```diff
@@ -0,0 +1,181 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <string>
+#include <utility>
+
+#include "base/bind.h"
+#include "base/strings/stringprintf.h"
+#include "brave/components/brave_rewards/browser/database/database_recurring_tip.h"
+#include "sql/statement.h"
+#include "sql/transaction.h"
+
+namespace brave_rewards {
+
+namespace {
+  // TODO(nejczdovc): rename to recurring_tip
+  const char* table_name_ = "recurring_donation";
```

Comment:
non member variables do not need an underscore at the end of the name

Reply:
nit: `const char table_name[] = "recurring_donation";`

Reply:
covered in https://github.com/brave/brave-browser/issues/7145